Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,32 @@
 import React from 'react';
 import './Button.css';
+
+class ActionButton extends React.Component {
+    constructor () {
+        super()
+        this.state={}
+    }
+
+    render(props) {
```

Comment:
No need to pass props into render. Props are already built into React classes. We typically target them by using `this.props`

Reply:
Thank you Clara! I will avoid doing that and try to be more conscious of the semicolons!